docs/root/version_history/current.rst

```diff
@@ -1,14 +1,47 @@
 1.15.0 (Pending)
 ================
 
-Changes
+
+Incompatible Behavior Changes (things that expected to cause an incompatibility if they apply; changes required)
+-----------------------------
+
+* build: official released binary is now built on Ubuntu 18.04, requires glibc >= 2.27.
+
+Minor Behavior Changes (things that may cause issues for some users, but should not for most)
+----------------
+
+* access loggers: applied existing buffer limits to access logs, as well as :ref:`stats <config_access_log_stats>` for logged / dropped logs. This can be reverted temporarily by setting runtime feature `envoy.reloadable_features.disallow_unbounded_access_logs` to false.
+* http: fixed several bugs with applying correct connection close behavior across the http connection manager, health checker, and connection pool. This behavior may be temporarily reverted by setting runtime feature `envoy.reloadable_features.fix_connection_close` to false.
```

Comment:
So do we want to dup issues like this to the bugfix section since they're both, or document (maybe in CONRIBUTING) that when changes overlap sections we add it to the first (as most important)?

Reply:
I think we'll have to iterate on this going forward. My current thought is let's put it into the first/most-important section.